sstables/storage.cc

```diff
@@ -583,6 +583,9 @@ sstring s3_storage::make_s3_object_name(const sstable& sst, component_type type)
     if (!sst.generation().is_uuid_based()) {
         throw std::runtime_error("'S3' STORAGE only works with uuid_sstable_identifier enabled");
     }
+    if (sst.is_uploaded()) {
+        return format("/{}/{}", _bucket, sst.filename(type));
```

Comment:
Note: to avoid limits on S3 "prefix" ops, we may need to have an sstable-per-prefix. But that can be done much later, and we'll discover which format we happen to use.

Reply:
Sure